numpy/core/function_base.py

```diff
@@ -52,8 +52,10 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None,
         If True, return (`samples`, `step`), where `step` is the spacing
         between samples.
     dtype : dtype, optional
-        The type of the output array.  If `dtype` is not given, infer the data
-        type from the other input arguments.
+        The type of the output array.  If `dtype` is not given, the data type
+        is inferred from `start`, `stop`, and `num`. The inferred dtype will
```

Comment:
I would leave `num` off the list - it must be an integer, but for the purposes of determining the dtype it is cast to float internally.

Reply:
Agreed, `num` is dont-care.